### DISCUSSION TITLE
Add a warning when running in windows compatibility mode

### DISCUSSION THREAD

**Discussion by Natelytle** [ppy/osu#21803] [Ideas] [answered]
I was troubleshooting someone's lazer performance in a discord server I am in (>50fps, lots of frame spikes) and it had turned out they were running the game in Windows 98/ME compatibility mode and had no idea. I think adding a warning for this similar to running with audio disabled or the low battery warning could be good for UX when people's games are running badly because of it.

Before:
![image](https://user-images.githubusercontent.com/92956514/209580281-597837e2-25a0-4b32-82ab-a896f4e4a8ee.png)
![image](https://user-images.githubusercontent.com/92956514/209580291-6edd0528-4f65-4e53-85fb-7a559b729d99.png)

After:
![image](https://user-images.githubusercontent.com/92956514/209580313-52f7f404-2322-4c83-9b96-9f57293f5984.png)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/22092
